src/renderer/store.js

```diff
@@ -0,0 +1,90 @@
+import { create } from 'zustand'
+import { persist, createJSONStorage } from 'zustand/middleware'
+import store from '../persist-store'
+
+/**
+ * @type {import('zustand/middleware').StateStorage}
+ */
+const storage = {
```

Comment:
We should probably set a [schema to storage](https://github.com/sindresorhus/electron-store#schema)

Reply:
Not sure what that would look like or how best implement it... We can chat about that later though